volestipy/volestipy/src/bindings.cpp

```diff
@@ -77,6 +80,9 @@ double HPolytopeCPP::generate_samples(int walk_len, int number_of_points,
                                       bool ball_walk, double a, double L, bool max_ball,
                                       double* inner_point, double radius, double* samples){
    
+   
+   std::cout<<"A = "<<HP.get_mat()<<"\n b= "<<HP.get_vec()<<std::endl;
```

Comment:
do you want to print this always when generate_samples is called?

Reply:
All cout statements have been removed